link/kprobe.go

```diff
@@ -49,6 +49,8 @@ type KprobeOptions struct {
 	//
 	// Needs kernel 5.15+.
 	Cookie uint64
+	// Kprobe symbol offset.
```

Comment:
This needs more context. How is it used? Why is it necessary? How would the caller determine the correct offset to use?

Reply:
Hi, This offset is used for kprobe, because some functions in kernel is static or inline, we can't insert one probe. but we can use probe the caller with offset. This "offset" is same as tracefs kprobe_events

https://www.kernel.org/doc/html/latest/trace/kprobetrace.html

Reply:
Hi, This offset is used for kprobe, because some functions in kernel is static or inline, we can't insert one probe. but we can use probe the caller with offset. This "offset" is same as tracefs kprobe_events

https://www.kernel.org/doc/html/latest/trace/kprobetrace.html

Reply:
```suggestion

	// Offset of the kprobe relative to the traced symbol.
	// Can be used to insert kprobes at arbitrary offsets in kernel functions,
	// e.g. in places where functions have been inlined.
```